### PR TITLE
Bug 1126378 - fix display of arrow of ICE contacts in the Emergency dialer

### PR DESCRIPTION
**Pull request by autra**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[autra Augustin Trancart (autra)](https://github.com/autra) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=020ab62cc466)

File: apps/emergency-call/index.html [hunk 1]
```diff
@@ -60,8 +60,9 @@
           <button id="keypad-delete" class="reset-button" data-value="delete" data-l10n-id="keypadDelete">
           </button>
         </div>
-        <section id="ice-contacts-bar" class="ice-contacts-bar" hidden data-l10n-id="ice-contacts-bar-text">
-          ICE Contacts - In Case of Emergency
+        <section id="ice-contacts-bar" class="ice-contacts-bar" hidden>
+          <span class="placeholder"></span>
```
Comment (gabrielesvelto) [inline apps/emergency-call/index.html]: The class name is a little too generic for my tastes. I'd rather have a dedicated id for this with a self-explanatory name such as `ice-contacts-bar-arrow`.

File: apps/emergency-call/style/ice_contacts.css [hunk 2]
```diff
@@ -17,8 +17,19 @@
   color: #FFF;
   line-height: 4.5rem;
   padding: 0 1.6rem 0 1.5rem;
+}
+
+.ice-contacts-bar > span.placeholder {
```
Comment (gabrielesvelto) [inline apps/emergency-call/style/ice_contacts.css]: Make this selector use the `ico-contacts-bar-arrow` id as describe above. Then there will be less chances of class name clash with shared CSS.

File: apps/emergency-call/style/ice_contacts.css [hunk 3]
```diff
@@ -17,8 +17,19 @@
...
+  height: 100%;
+  position: absolute;
+  left: 0;
   background-image: url('images/white_arrow.png');
   background-repeat: no-repeat;
   background-position: 28.8rem center;
   background-size: 2.4rem;
 }
+
+html[dir=rtl] .ice-contacts-bar > span.placeholder {
```
Comment (gabrielesvelto) [inline apps/emergency-call/style/ice_contacts.css]: Likewise, use the `ice-contacts-bar-arrow` id instead.

**Comment by try-server-hook** [comment]
[autra Augustin Trancart (autra)](https://github.com/autra) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=47935f15ecfb)

**Pull request merged**